src/Items/ItemShears.h

```diff
@@ -90,6 +90,14 @@ class cItemShearsHandler :
 			}
 		}
 	}
+
+
+
+	virtual float GetBlockBreakingStrength(BLOCKTYPE a_Block)
+	{
+		return !IsBlocksWeb(a_Block) && !IsBlockMaterialLeaves(a_Block) ? (IsBlocksWool(a_Block) ? 5.0f/* WOOL */ : super::GetBlockBreakingStrength(a_Block) /* none of them */) : 15.0f /* WEB and LEAVES */;
```

Comment:
This is still too complicated, use proper `if` statements instead of the ternary operator.